src/xdocs/config_misc.xml

```diff
@@ -2490,10 +2490,64 @@ public record MyRecord1 {
           To configure the check:
         </p>
         <source>
+&lt;module name=&quot;UniqueProperties&quot;/&gt;
+        </source>
+        <p>
+          Example: in foo.properties file
+        </p>
+        <source>
+key.one=44
+key.two=32 // OK
+key.one=54 // violation
+        </source>
+        <p>
+            To configure the check to scan custom file extensions:
+        </p>
+        <source>
+&lt;module name=&quot;UniqueProperties&quot;&gt;
+  &lt;property name=&quot;fileExtensions&quot; value="customproperties"/&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+            Example: in foo.customproperties file
+        </p>
+        <source>
+key.one=44
+key.two=32 // OK
+key.one=54 // violation
+        </source>
+        <p>
+            Example: in foo.properties file
+        </p>
+        <source>
+key.one=44
+key.two=32 // OK
+key.one=54 // OK
+        </source>
+        <p>
+            To configure the check to scan custom file extensions and properties file:
+        </p>
+        <source>
 &lt;module name=&quot;UniqueProperties&quot;&gt;
-  &lt;property name=&quot;fileExtensions&quot; value=&quot;properties&quot; /&gt;
+  &lt;property name=&quot;fileExtensions&quot; value="properties, customproperties"/&gt;
 &lt;/module&gt;
         </source>
+        <p>
+            Example: in foo.customproperties file:
+        </p>
+        <source>
+key.one=44
+key.two=32 // OK
+key.one=54 // violation
+        </source>
+        <p>
+            Example: in foo.properties file:
```

Comment:
please fix indentation here and above. It should be 2, not 4 spaces

Reply:
@strkkk I could not understand completely what changes you suggested for the examples. Currently we have 3 examples, should I make it 2? Also, `properties` and `PROPERTIES` would be different, so did you mean to add `PROPERTIES` in place of `customproperties`, in case of which `foo.properties` will have `// OK, file is not checked` when `fileExtension="PROPERTIES"`.
If you could please clear my ambiguity.
Thanks for suggesting the changes!

Reply:
I see that now everything is ok, so I guess you got it correctly :)